### PR TITLE
Enforce consistent formatting of sigs.yaml

### PR DESCRIPTION
**Pull request by spiffxp**
This is intended to lay the groundwork for better validation of
sigs.yaml, enforcement/autogeneration of conventions, and idemptotently
regenerating sigs.yaml from sharded files.

This strips away any existing comments in sigs.yaml.  I tried to keep
them, but found it wasn't worth it given the number of comments
currently in place.

I tried using python and ruamel.yaml, but that would leave our
formatting/domain validation in two places.

I tried using gopkg.in/yaml.v3, but couldn't find a clean way use both
the `yaml.Node` object as well as the actual structs in question.

See individual commits for details

/sig contributor-experience

### CODE REVIEW

**Comment by cblecker** [comment]
/lgtm
/approve
/hold

This all looks reasonable to me.

File: committee-code-of-conduct/README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ The Kubernetes Code of Conduct Committee (CoCC) is the body that is responsible
 
 ## Members
 
-* Jaice Singer Dumars (**[@jdumars](https://github.com/jdumars)**), Google
-* Paris Pittman (**[@parispittman](https://github.com/parispittman)**), Google
+* Jennifer Rondeau (**[@bradamant3](https://github.com/bradamant3)**), VMware
```
Comment (nikhita) [inline committee-code-of-conduct/README.md]: non-blocking comment, can be moved to another PR: should be s/VMware/Stripe here.

Ref: https://github.com/kubernetes/community/pull/3523#discussion_r270214887

**Comment by spiffxp** [comment]
/hold cancel
I’ll correct in a follow up. Want to avoid rebase fun
